less/mixins.less

```diff
@@ -1 +1,5 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
+.button-layout(@background-color; @color) {
+    background-color: @background-color;
+    color: @color;
+}
```

Comment:
Task 4: Desktop Updates Needed - WELL DONE!
 Navigation: Use the navigation.less file for all your navigation styling - DONE!
 Main Header: Use the home-page.less file for the header styling. DONE!
 Buttons: Create a parametric mixin that can create the missing buttons in the design file. Use the mixins.less file to create your mixin. - DONE!